client/src/tun.rs

```diff
@@ -1 +1,44 @@
+mod tun {
+    static ACC_MTU_SIZE: i32 = 1280;
+    use tun::{
+        platform::{linux, Device},
+        IntoAddress,
```

Comment:
If IntoAddress is a Tun specific trait we shouldn't use it directly. Need to keep up our IPv6 support which Rust Tun doesn't default nicely to